### PR TITLE
Dev 1.0

### PR DESCRIPTION
**Pull request by erllypaguntalan**

### CODE REVIEW

File: File_F.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+Forever
```
Comment (maragatchalian) [inline File_F.txt]: Change this to
F
O
R
E
V
E
R

File: File_C.txt [hunk 2]
```diff
@@ -0,0 +1 @@
+I love
```
Comment (maragatchalian) [inline File_C.txt]: Change this to your complete name

File: File_D.txt [hunk 3]
```diff
@@ -0,0 +1 @@
+you!
```
Comment (maragatchalian) [inline File_D.txt]: Change this to the name of your school

File: File_E.txt [hunk 4]
```diff
@@ -0,0 +1 @@
+Walang
```
Comment (maragatchalian) [inline File_E.txt]: Please change this to "KLab Cyscorpions" :bow:

File: File_F.txt (hunk 1 above)
Comment (erllypaguntalan) [inline File_F.txt]: Done!

File: File_C.txt (hunk 2 above)
Comment (erllypaguntalan) [inline File_C.txt]: Done!

File: File_D.txt (hunk 3 above)
Comment (erllypaguntalan) [inline File_D.txt]: Done!

File: File_E.txt (hunk 4 above)
Comment (erllypaguntalan) [inline File_E.txt]: Done!

**Pull request merged**